future_webpage/dgf/fixtures/pdga_divisions.json

```diff
@@ -0,0 +1,338 @@
+[
```

Comment:
este archivo lo he sacado convirtiendo los datos que hay aquí: https://www.pdga.com/pdga-documents/tour-documents/divisions-ratings-and-points-factors a mano

Este tipo de json es un fixture de Django: https://docs.djangoproject.com/en/3.0/howto/initial-data/

Es una cosa pa poner datos iniciales en base de datos. No he encontrado un sitio como los discos para leer las divisiones de forma automáticas de la PDGA... por lo que lo que he hecho ha sido meter este archivo con las divisiones que existen ahora mismo.

Esto es lo mismo que meterte en el server e ir creando las divisiones a mano. Conclusión: se podría hacer algo así tb para meter los friends pero bueno ya los ha metido Mario a mano..

Reply:
el meter los fixtures en DB sería una cosa inicial, pasa una vez y nunca más. Se hace con un comando de Django:

`manage.py loaddata <fixturename>`

Reply:
¿Qué pasa si lo haces más de una vez? (imagino que ignorará los datos duplicados y/o lo sobrescribirá ¿no?) 

Estoy pensando si no es buena idea meterlo en `/ci/server_restart.bash` como comando extra, así no se nos olvida en ningún lao... y también en el `run.bash` para los tests y tal... por cierto... ¿Y los tests de esto?

Reply:
- Si haces esto más de una vez los reescribe (el primary key es el id), así que no pasa na
- Lo de meterlo en el restart... no se, no lo veo porque no es una cosa que queramos ejecutar siempre, es hacer el restart más lento pa na creo yo. Esto en verdad es una feature de django para importar cosas. Como he dicho antes: esto es lo mismo que meterlos a mano en el admin así que no creo que tenga sentido automatizar pa na.
- El tema de los tests: sí, pero yo no lo haría automático. Si el test lo necesita, pos que lo cargue: https://docs.djangoproject.com/en/3.0/topics/testing/tools/#topics-testing-fixtures
- ¿Por qué no hay tests para cargar los fixtures? Porque es una feature de Django